generation/maisi/maisi_diff_unet_training_tutorial.ipynb

```diff
@@ -57,7 +57,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    display(Image(filename="./figures/maisi_diff-train-examples.png"))
</pre><p>Could you please also include the data in the pr, otherwise the ci will failed, thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1860/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Thank you. I just included the figure in the PR.